azure-pipelines.yml

```diff
@@ -280,3 +280,31 @@ jobs:
       inputs:
         pathtoPublish: '$(Build.ArtifactStagingDirectory)/SymbolPackages'
         artifactName:  Microsoft.Spark.Binaries
+
+- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
```

Comment:
do you need a separate job? Can't you do this in a previous job?

Reply:
Unfortunately it seems the previous job's image doesn't have bash.  I think it is possible to use the previous job, but it would require more steps as well as hard coding the version / runtime / framework values.

Reply:
I'll look into this a bit more.

Reply:
https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/bash?view=azure-devops wouldn't work?

Reply:
You wouldn't need this condition again, since it is already defined above before the package job. So by just adding the right indentation to this job, they both would share the same condition.

Reply:
Removed the separate job